client/src/components/ImageViewer.vue

```diff
@@ -16,22 +42,57 @@ import exifr from "exifr/dist/full.esm.mjs";
 import { ref, reactive } from "vue";
 import emitter from "tiny-emitter/instance";
 import { config } from "../config";
+import { PencilAltIcon } from "@heroicons/vue/outline";
+import axios from "axios";
 
 export default {
+  components: { PencilAltIcon },
   setup() {
     let newImgUrl = ref("");
     let imgTags = reactive({});
+    const editMode = ref(false);
+    const editUrl = ref("");
     async function parseImageTags(url) {
       imgTags.value = await exifr.parse(url, {
-        iptc: true,
-        pick: [...config.imgTags],
+        iptc: { pick: [...config.imgTags] },
+        exif: { pick: [...config.imgTags] },
       });
     }
-    emitter.on("updateImageViewer", function (url) {
+    function composeImageFolderPath(path) {
+      let newPath = path.split("/");
+      return newPath[2];
+    }
+    function setImageFolderPath(path) {
+      editUrl.value = config.baseUrl + "/edit/" + composeImageFolderPath(path);
+    }
+    emitter.on("updateImageViewer", function (url, path) {
       newImgUrl.value = url;
       parseImageTags(url);
+      setImageFolderPath(path);
     });
-    return { parseImageTags, newImgUrl, imgTags };
+    async function submitEdits(submitEvent) {
+      /* get form values */
+      const params = Object.fromEntries(
+        new FormData(submitEvent.srcElement).entries()
+      );
+      /* send form values via query params e.g: /edit/artefaktId?param=value */
+      try {
+        const url = editUrl.value;
+        const response = await axios.post(url, null, { params });
+        console.log(response);
+        editMode.value = false;
+      } catch (error) {
+        console.log(error);
```

Comment:
Vielleicht hier console.error anstatt einfach logging nutzen. (Optional)

Und als Sidenote - super Video zu console-logging :)
https://www.youtube.com/watch?v=L8CDt1J3DAw

Reply:
Guter Einwand. Video kenn ich ✌️
In Produktion doch eigentlich eh keine console.logs oder was sagst du?